### PR TITLE
Delete documentation about canonical data

### PR DESCRIPTION
**Pull request by kytrinyx**
This was about how to extract canonical data. The [improving consistency across tracks](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/improving-consistency-across-tracks.md) document now covers both extracting and using canonical data.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -385,25 +384,6 @@ A problem must have a unique slug. This slug is used as
...
-  * Search for the exercise in this list http://x.exercism.io/v3/problems/data-todos (your browser might need a JSON viewer plugin/add-on to show this)
-  * Run this command in your terminal `curl http://x.exercism.io/v3/problems/data-todos | jq '.[] | map(select(.["slug"] == "accumulate"))'`. Replace `accumulate` with the name of the exercise you have chosen.
-3. Open the links in different browser tabs, navigate to the test files and read through the tests. Look for test cases that all/many of the languages have in common and identify outliers that only appear in one or two languages. It might be helpful to make some notes on your findings.
-4. Now it's time to write the canonical-data.json file. Look at examples like [the test data for hamming](https://github.com/exercism/x-common/blob/master/exercises/hamming/canonical-data.json) and check the [readme](https://github.com/exercism/x-common#test-data-format-canonical-datajson) to find out how to best structure the JSON file.
-From what you learned in step 3 decide for a sensible set of test cases. The following considerations might help with that.
```
Comment (Insti) [inline CONTRIBUTING.md]: These first 5 steps seem to not be covered in the linked replacement document.

Comment (kytrinyx) [inline CONTRIBUTING.md]: I left them off because they're not necessary anymore.

Comment (kytrinyx) [inline CONTRIBUTING.md]: To clarify: We link to the open issues, and the open issues has a link to a dynamic list (always up-to-date) of the tracks that have implemented the exercise:

https://github.com/exercism/x-common/issues/592

Comment (Insti) [inline CONTRIBUTING.md]: Ok. 

Another complication: That issue (and presumably all the other related ones) references this section of the document, so those links will break?

Comment (kytrinyx) [inline CONTRIBUTING.md]: Oooh. That's a great point. Yeah... so let's go ahead and keep this section (stick it at the bottom of the document), but only have one line that points to the documentation in the docs repo.

**Comment by kytrinyx** [comment]
Alright, I've given this another go.

**Comment by Insti** [comment]
Github is showing this has merge conflicts, but apart from that I think it's OK now.

**Comment by kytrinyx** [comment]
Oh, right. I merged another PR in the meantime. Fixed, and merging.

**Pull request merged**